package.json

```diff
@@ -64,7 +64,7 @@
     "vite-plugin-environment": "^1.1.3",
     "vitest": "^0.33.0",
     "vue-class-component": "^7.2.6",
-    "vue-plugin-helper-decorator": "^0.0.12",
+    "vuelidate-property-decorators": "1.0.28",
```

Comment:
Is this needed for Storybook?

Reply:
No, I don't think so! It's needed to use the `@Validations` property decorator.

Reply:
Actually, I take that back. It is indeed needed for Storybook. I tried running Storybook and looking at BaseAddress, it wasn't working since it failed importing the module. Sorry about that.

Reply:
Interesting that SB needs it since it's actually a component dependency. Did you update the Lerna dependencies (`lerna bootstrap --hoist`)?

Reply:
I did not actually. Totally forgot. You're 100% right Sev. It is not needed by SB.